proof/refine/AARCH64/ArchStateRelation.thy

```diff
@@ -128,12 +129,13 @@ definition ghost_relation ::
 (* FIXME arch-split: provided only so that ghost_relation can be used within generic definition
    of state_relation (since arch_state_relation doesn't have access to kheap, and
    gsPTTypes on AARCH64 isn't generic) *)
```

Comment:
It's unclear to me if this FIXME is still needed. Should I remove it, or do we think that we might revisit the idea of the ghost_relation_wrapper?

Reply:
Cool, then let's do that (drop FIXME, keep comment)